run_ocsci.py

```diff
@@ -58,13 +57,6 @@ def init_ocsci_conf(arguments=[], default_config=OCSCI_DEFAULT_CONFIG):
         with open(cluster_config) as file_stream:
             cluster_config_data = yaml.safe_load(file_stream)
         update_dict_recursively(config_data, cluster_config_data)
-    rendered_config = render_yaml_with_j2_context(config_data)
-    for key, value in rendered_config.items():
-        if key == "DEFAULTS":
-            for default_key, default_value in value.items():
-                setattr(defaults, default_key, default_value)
-        elif key.isupper():
```

Comment:
This update config, so I guess you don't want to delete this one Vasu

Reply:
Ahh, Thanks, just need to skp the defaults part!